PortalFetch/crawler_download.py

```diff
@@ -3,277 +3,246 @@
 
 It requires file 'user.ini' to load the user's own user name and password.
 """
+import sys
+import time
+import logging
+import chromedriver_binary
+import datetime
 from configparser import ConfigParser
 from selenium import webdriver
 from selenium.webdriver.common.action_chains import ActionChains
 from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.support.ui import Select
 from selenium.common.exceptions import NoSuchElementException
-import sys
-import time
-import chromedriver_binary
-import logging
-from PortalFetch.crawler_data_process import DataProcess
-
-
-def submitClick(driver):
-    """
-    Search and click sumbit button in the search by term page.
-
-    Parameters: webdriver
-    Returns: None
-    """
-    inputs = driver.find_elements_by_tag_name("input")
-    submit = ""
-    for input in inputs:
-        if "submit" == input.get_attribute("type") and input.is_enabled() and input.is_displayed():
-            submit = input
-            break
-    submit.click()
-    time.sleep(2)  # Waiting for completion
-
-
-def advanceSearch(driver):
-    """
-    Locate 'Advanced Search' button and click in the 'Look Up Classes' page.
+from selenium.common.exceptions import ElementNotVisibleException
+from crawler_data_process import DataProcess
+
+logging.basicConfig(filename = '../log/' + 
```

Comment:
maybe we should create logger package in general. Then we just need to do pip install..